bindata/v3.11.0/kube-apiserver/pod.yaml

```diff
@@ -13,10 +13,21 @@ spec:
     - name: fix-audit-permissions
       image: ${IMAGE}
       imagePullPolicy: IfNotPresent
-      command: ['sh', '-c', 'chmod 0700 /var/log/kube-apiserver']
+      command: ['/bin/sh', '-c', 'chmod 0700 /var/log/kube-apiserver']
       volumeMounts:
         - mountPath: /var/log/kube-apiserver
           name: audit-dir
+    - name: wait-for-port
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      command: ['/usr/bin/timeout', '90', "/bin/bash", "-c"]
+      args:
+      - |
+        echo -n "Waiting for port :6443 to be released."
+        while [ -n "$(lsof -ni :6443)" ]; do
+          echo -n "."
```

Comment:
90 dots in my log :-)

Reply:
90 should be enough for everybody. I can put in a newline after 72 if you like ;-)